recipes/xnnpack/all/conanfile.py

```diff
@@ -63,7 +63,8 @@ def layout(self):
     def requirements(self):
         self.requires("cpuinfo/cci.20220228")
         self.requires("fp16/cci.20210320")
-        self.requires("pthreadpool/cci.20210218")
+        self.requires("pthreadpool/cci.20210218", transitive_headers=True)
+        self.requires("fxdiv/cci.20200417")
```

Comment:
```suggestion
```

Reply:
It's a mistake in their CMakeLists, you can check their source code (I'm pretty sure to have carefully checked this when I've created xnnpack recipe).

Reply:
@jhabermas In CMakeLists wrapper along conanfile.py, there is comment regarding fxdiv, could you remove it please (just the comment)? Because it's misleading.